core/src/main/scala/clue/GraphQLException.scala

```diff
@@ -14,4 +14,5 @@ case class DisconnectedException() extends GraphQLException("Connection was clos
 case class InvalidSubscriptionIdException(id: String)
     extends GraphQLException(s"Invalid subscription id: $id")
 
-case class ResponseException(errors: GraphQLErrors) extends GraphQLException(errors.toString)
+case class ResponseException[D](errors: GraphQLErrors, data: Option[D])
```

Comment:
Will this be used for the general case when there are some errors and some data?

Reply:
Right, this gives access to the `data` (if present) when `ErrorPolicy.RaiseAlways` is used and there are `errors` in the response.